services/dialog/v1.js

```diff
@@ -146,31 +146,31 @@ module.exports = function (RED) {
           params.dialog_id = dialogid;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 space characters but found 10.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2462016486/issues/source?bid=3225115&fileBranchId=3255594#l147)